pkg/controller/job/job_controller.go

```diff
@@ -150,6 +159,7 @@ func newControllerWithClock(podInformer coreinformers.PodInformer, jobInformer b
 		broadcaster:           eventBroadcaster,
 		recorder:              eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "job-controller"}),
 		clock:                 clock,
+		backoffInfoForJob:     make(map[string]backoffInfo),
```

Comment:
We need to clean the map for deleted jobs.

Reply:
Done